charts/fulcio/values.schema.json

```diff
@@ -513,6 +513,26 @@
                                 "fileca"
                             ]
                         },
+                        "kms_resource": {
+                            "$id": "#/properties/server/properties/args/properties/kms_resourse",
+                            "type": "string",
+                            "title": "The kms resource schema",
+                            "description": "The KMS URI for backend KMS",
+                            "default": "",
+                            "examples": [
+                                "gcpkms://foo/bar"
```

Comment:
Can we give the examples for the other backends too, azurekms, hashivault, and awskms?